hr_timesheet_sheet_invoice_approved/views/hr_analytic_timesheet_view.xml

```diff
@@ -19,7 +19,7 @@
             <field name="arch" type="xml">
                 <xpath expr="//filter[@name='others']" position="after">
                     <separator orientation="vertical"/>
-                    <filter name="approved_timesheet" string="Approved Timesheet" domain="['|', ('is_timesheet_line','=', False), ('sheet_approved', '=', True)]"/>
+                    <filter name="approved_timesheet" string="Approved Timesheet" domain="['&amp;', ('is_timesheet_line','=', True), ('sheet_approved', '=', True)]"/>
```

Comment:
You're right but I like beeing verbose into my domain declaration because I think it is more easier to read and even more if the domain become more complicated.